pkg/session_recording/terminal.jsx

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
Could you make a more descriptive commit message? I know already what's going on, but it's better to get used to explaining for people who don't.

Something like "Copy cockpit terminal component for expanding' could be a bit better.